README.md

```diff
@@ -6,7 +6,7 @@ Python Version of [Persian.js](https://github.com/itmard/persian.js)
 
 ###Install Package
 ```
-pip install persian
+pip3 install persian
```

Comment:
Why changed this ? the official tool name is pip (specially using a virtualenv ) people can change it to other aliases like pip3